CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 0.3.0 (Thaler Testnet v0.5)
+
+### New features
+
+- Introduce `toUnsignedHex()` method to staking account related transaction builders
+- Introduce `toSignedPlainHex()` method to`WithdrawUnbondedTransactionBuilder`
+- Introduce `compressedPublicKey` property to`KeyPair`
+
+### Breaking Changes
+
+- `cro.network.Devnet()` now requires fee configuration
+- `cro.network.fromChainId()` now returns the chain enum instead of network configuraiton
```

Comment:
small typo with the "configuration" spelling

Reply:
fixed, thanks for spotting that.